deploy/serverpc/litpose/run_litpose.py

```diff
@@ -3,7 +3,9 @@
 # import cProfile
 # import pstats
 
-from iblvideo import lightning_pose, download_lit_model
+from iblvideo import download_lit_model
+from iblvideo.pose_lit import lightning_pose
```

Comment:
we don't want to import `lightning_pose` from `__init__.py` if we're in the dlc environment